src/oncall/api/v0/__init__.py

```diff
@@ -71,3 +71,7 @@ def init(application, config):
 
     from . import upcoming_shifts
     application.add_route('/api/v0/users/{user_name}/upcoming', upcoming_shifts)
+
+    if config['iris_plan_integration']['activated']:
+        from . import team_escalate
+        application.add_route('/api/v0/teams/{team}/escalate', team_escalate)
```

Comment:
I suggest changing this to `iris_escalate` since it's not a generic escalate endpoint.